lib/signaling/v2/peerconnection.js

```diff
@@ -389,7 +388,9 @@ PeerConnectionV2.prototype._maybeReoffer = function _maybeReoffer(offer) {
   // re-introducing 'offerToReceiveAudio' to the default RTCOfferOptions with a
   // value > 1.
   if (isFirefox) {
-    var senders = this._peerConnection.getSenders();
+    var senders = this._peerConnection.getSenders().filter(function(sender) {
+      return sender.track;
+    });
```

Comment:
Why do we need to filter here?

Reply:
@markandrus Because the spec for getSenders() says that there might be senders whose tracks are null.

Reply:
Right, but is this a problem for the way we use this collection? I think we want to ensure that there is at least an RTCRtpSender available—not necessarily that it has a `track` present. Perhaps it's also sufficient to check for RTCRtpReceivers, since there should be the same number.

Reply:
@markandrus But aren't we basing a decision to reoffer based on the number of senders. So shouldn't we count only those senders which are active, ie whose track is not null?

Reply:
Ah, OK—yeah, thinking about it some more, the logic here is intended to ensure we can match up MediaStreamTracks to m=lines, so we should be checking RTCRtpSenders with non-null `track` properties. 👍